docs/editor_manual/administrator_tasks/managing_users.md

```diff
@@ -4,8 +4,6 @@ As an administrator, a common task will be adding, modifying or removing user pr
 
 This is done via the 'Users' interface, which can be found in the Settings menu, accessible via the left-hand menu bar.
 
-![](../../_static/images/screen35_users_menu_item.png)
```

Comment:
Removed because it just shows the menu item in the sidebar, which doesn’t feel very useful.